README.md

```diff
@@ -1,9 +1,26 @@
-# OpenShift Installer
+# KNI Installer
```

Comment:
Restoration of README and addition of IPI docs is handled here: https://github.com/openshift-metal3/kni-installer/pull/89